pymc4/inference/utils.py

```diff
@@ -53,7 +53,11 @@ def initialize_sampling_state_smc(
         The list of names of the model's deterministics
     """
     eval_func = flow.evaluate_model_smc
-    _, state = eval_func(model, observed=observed, state=state, num_chains=smc_draws)
+    _, state = eval_func(model, observed=observed, state=state, draws=smc_draws)
+    print("FUCKKKKKKKKKKKkkkk")
```

Comment:
I LOLed.

Reply:
Sorry, I thought it was still there) it is resolved